src/test/java/com/artipie/api/RestApiServerBase.java

```diff
@@ -140,10 +164,14 @@ final void beforeEach(final Vertx vertx, final VertxTestContext context) throws
         final InMemoryStorage storage = new InMemoryStorage();
         this.asto = new BlockingStorage(storage);
         this.caches = new SettingsCaches.Fake();
+        if (this.keyStore().isPresent()) {
+            this.initKeyStore();
```

Comment:
You can do an initialization routine inside `keyStore()`.

Reply:
@olenagerasimova @dgarus Thank you for advices, please look at changes.